lib/src/cupertino/cupertino_popover_menu.dart

```diff
@@ -328,6 +342,11 @@ class RenderPopover extends RenderShiftedBox {
 
   @override
   bool hitTest(BoxHitTestResult result, {required Offset position}) {
+    final borderPath = _buildBorderPathForOffset(position);
```

Comment:
If this path is needed for hit testing, then we should create it during layout and cache it, rather than re-create it here.